# file: C:\Users\Couch\AppData\Roaming\Python\Python313\site-packages\fastapi\security\open_id_connect_url.py
# hypothesis_version: 6.141.1

['Authorization', 'Not authenticated']